ITISFoundation/osparc-simcore
# qxapp to osparc name refactor

# Potentially breaking changes. This includes a change in every file of the frontend, plus some configuration files.
I have tested that the application builds and runs correctly. I have also tested that it works fine, but it will probably create conflicts with the current merge requests.


## What do these changes do?


Remove the old "qxapp" application name and use "osparc" instead.

## How to test


* Build the application and see that everything runs properly.

  * Try to create new studies, modify them..
  * Assets like images are retrieved properly
  * Testing and deployment still works
